### DISCUSSION TITLE
Conceptual figures to visualize processes

### DISCUSSION THREAD

**Discussion by Jan-MarkusHomberger** [openearth/aeolis-python#210] [General]
Hi everyone,

I saw that you started updating the AeoLiS documentation. As a new user, I was sometimes struggling in understanding what each process is supposed to do. What do you think about supporting the process documentation with conceptual figures similar to what you  did for the hydrological process? ![Capture](https://github.com/user-attachments/assets/f4356a4f-3046-498f-b67c-acc4bfe25842)

**Comment by christavanijzendoorn** [comment]
Hi Jan-Markus, I've just restructured the model description section and I agree that there is not a lot of visual supporting information. Would you want to help me make a list of figures that would be helpful? Then we can specifically (ask people to) add those figures.
